UnityEngine.AssetBundles/Editor/Tests/ABModelTests.cs

```diff
@@ -45,25 +45,43 @@ public void TearDown()
     [Test]
     public void AddBundlesToUpdate_AddsCorrectBundles_ToUpdateQueue()
     {
+        // Account for existing asset bundles
+        int numBundles = ABModelUtil.BundlesToUpdate.Count;
+
         Model.AddBundlesToUpdate(m_BundleInfo);
-        Assert.AreEqual(3, ABModelUtil.BundlesToUpdate.Count);
+        Assert.AreEqual(numBundles + 3, ABModelUtil.BundlesToUpdate.Count);
     }
 
     [Test]
     public void Update_ReturnsTrue_ForRepaintOnFinalElement()
     {
+        // Account for existing bundles
+        int numChildren = ABModelUtil.Root.GetChildList().Count;
+
         Model.AddBundlesToUpdate(m_BundleInfo);
-        Assert.IsFalse(Model.Update());
-        Assert.IsFalse(Model.Update());
-        Assert.IsTrue(Model.Update());
+
+        for (int i = 0; i < numChildren; ++i)
+        {
+            if (i < numChildren)
+            {
+                Assert.IsFalse(Model.Update());
+            }
+            else
+            {
+                Assert.IsTrue(Model.Update());
+            }
+        }
     }
 
     [Test]
     public void ModelRebuild_Clears_BundlesToUpdate()
```

Comment:
test should be renamed now that it is testing something slightly different (and more accurate). Maybe ```ModelRebuild_Resets_BundlesToUpdate_ToRootChildren```